scripts/data_preparation/generate_bicubic_img.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import cv2
+import glob
```

Comment:
This file should be in another PR.

One PR should be small and have a very clear function